lib/roadie/document.rb

```diff
@@ -29,12 +29,24 @@ class Document
     # Should CSS that cannot be inlined be kept in a new `<style>` element in `<head>`?
     attr_accessor :keep_uninlinable_css
 
+    # Should non-inlineable media queries be grouped?
+    # e.g.
+    # @media(max-width: 600px) { .col-6 { display: block; } }
+    # @media(max-width: 400px) { .col-12 { display: inline-block; } }
+    # @media(max-width: 600px) { .col-12 { display: block; } }
+    # will become
+    # @media(max-width: 600px) { .col-6 { display: block; } .col-12 { display: block; } }
+    # @media(max-width: 400px) { .col-12 { display: inline-block; } }
+    # which would change the styling on the page
+    attr_accessor :use_shared_media_queries
```

Comment:
Instead of "shared media queries", what do you think about "merge_media_queries"?